src/models/pull-request.ts

```diff
@@ -3,10 +3,23 @@ import Review from './review';
 export default class PullRequest {
     url: string;
     id: string;
-    reviews: Review;
+    reviews: Array<Review>;
 
-    constructor(id: string, url: string) {
+    constructor(id: string, url: string, reviews: Array<any>) {
```

Comment:
You've set `reviews` type to `Array<Review>` above, I believe you do want array of reviews in the constructor too.